ContentMetadata/ContentMetadata.cs

```diff
@@ -2,63 +2,69 @@
 
 namespace ContentMetadata
 {
-    [Serializable]
-    public class ContentMetadata
-    {
-        public ContentMetadata() { }
+	[Serializable]
+	public class ContentMetadata
+	{
+		public ContentMetadata() { }
 
-        public ContentMetadata(Guid contentId, Guid contentTypeId, string key, string value)
-        {
-            ContentId = contentId;
-            ContentTypeId = contentTypeId;
-            Key = key;
-            Value = value;
-        }
+		public ContentMetadata(Guid contentId, Guid contentTypeId, string key, string value, int userId = 0)
```

Comment:
I'm not sure why we need the int userId here? This is customer specific code and should be removed. Having the ContentId for the user should be enough to fetch it.